edx_lint/files/commitlint.config.js

```diff
@@ -28,4 +29,22 @@ module.exports = {
     // Those include the typical "Merged" messages, so those are implicitly ignored:
     // https://github.com/conventional-changelog/commitlint/blob/master/%40commitlint/is-ignored/src/defaults.ts
   ],
+
+  plugins: [
+    {
+      rules: {
+        'squash-match': ({subject}) => {
+          const SQUASH = 'squash';
+          return [
+            // matches squash commits, so use with "never"
+            // - Why does this match on empty type/subject?
+            // - Gives a false warning in this case, but not a big deal.
```

Comment:
**Note to Reviewer:** Using a commit with a subject, but no type at the beginning (e.g. "what is wrong with this commit"), the squash warning fires, but so does an empty subject error (which is not correct), in addition to the empty type error (which is correct).